test/test_view.c

```diff
@@ -54,23 +54,39 @@ int main(int argc, char *argv[])
 
     while ((c = getopt(argc, argv, "IbDCSl:t:i:o:N:BZ:@:")) >= 0) {
         switch (c) {
-        case 'S': flag |= 1; break;
+        case 'I': ignore_sam_err = 1; break;
```

Comment:
I changed the order to be consistent with the string passed to getopt, right above the switch statement. And for the usage instructions I aimed for what would make most sense for the user. Ideally they should all be consistent, but I didn't touch the getopt string because I'm not familiar with how it's used.